ios/RNMobileDeviceManager.m

```diff
@@ -0,0 +1,76 @@
+#import "RNMobileDeviceManager.h"
+
+// Used to send events to JS
+#if __has_include(<React/RCTBridge.h>)
+#import <React/RCTBridge.h>
+#elif __has_include("RCTBridge.h")
+#import "RCTBridge.h"
+#else
+#import "React/RCTBridge.h"
+#endif
+
+#if __has_include(<React/RCTEventDispatcher.h>)
+#import <React/RCTEventDispatcher.h>
+#elif __has_include("RCTEventDispatcher.h")
+#import "RCTEventDispatcher.h"
+#else
+#import "React/RCTEventDispatcher.h"
+#endif
+
+@implementation MobileDeviceManager
+
+@synthesize bridge = _bridge;
+
+static NSString * const appConfigurationKey = @"com.apple.configuration.managed";
+static NSString * const APP_CONFIG_CHANGED = @"react-native-mdm/managedAppConfigDidChange";
+
+- (instancetype)init
+{
+    [ManagedAppConfigSettings clientInstance].delegate = self;
+    [[ManagedAppConfigSettings clientInstance] start];
```

Comment:
I notice the framework creates an observer, but never tears it down. We'll need to make some edits to the swift file from the framework that lets us remove the observer